### PR TITLE
Fix the events parameters, optimize Export and change version to 2.3.6.2

### PR DESCRIPTION
**Pull request by hmoraes**
bugfix: Fix ImportCompleteArgs parameteres and ExportCompleteArgs TimeUsed
- Change version to 2.3.6.2

### CODE REVIEW

File: source code/MySqlBackup(Devart-Express)/EventArgs/ExportCompleteArgs.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public ExportCompleteArgs(DateTime timeStart, DateTime timeEnd, MySqlBackup.Proc
             _completionType = endType;
             _timeStart = timeStart;
             _timeEnd = timeEnd;
-            _timeUsed = timeStart - timeEnd;
+            _timeUsed = timeEnd - timeStart;
```
Comment (hmoraes) [inline source code/MySqlBackup(Devart-Express)/EventArgs/ExportCompleteArgs.cs:41]: Changed the order becase timeEnd is greater than timeStart

File: source code/MySqlBackup(Devart-Express)/MySqlBackup.cs [hunk 2]
```diff
@@ -1388,7 +1394,30 @@ void ReportEndProcess()
...
+                            break;
+                    }
+
+                    ImportCompleteArgs arg = new ImportCompleteArgs()
+                    {
+                        LastError = _lastError,
+                        CompletedType = completedType,
+                        TimeStart = timeStart,
+                        TimeEnd = timeEnd,
+                    };
```
Comment (hmoraes) [inline source code/MySqlBackup(Devart-Express)/MySqlBackup.cs:1420]: Populate ImportCompleteArgs with result data

**Comment by hmoraes** [comment]
I optimized the initial export backup process. At the beginning MySqlBackup reads a lot of metadata from tables to export. This process takes a long time when the database has too many tables. The change consisted in modifying the MySqlObjects to use a Dictionary instead of a List, because the List's Contains function is O(n) and in the Dictionary it is O(1)

**Comment by adriancs2** [comment]
Thank you very much, hmoraes
This is much appreciated.

**Pull request merged**